batch_uploads_imageuploader

```diff
@@ -169,7 +169,6 @@ while(my $line = <STDIN>)
 {
     chomp $line;
     my @linearray = split(" " , $line);
-    push (@resultsarray, @linearray);
     @fullpath = $linearray[0];
```

Comment:
Here, if only one line is given in STDIN, `@resultsarray` becomes:
0 'full path'
1 'is phantom'
2 'patient name'

hence, the `@resultsarray` length is 3 instead of 1

Reply:
Not sure I understand....Since `@fullPath` is initialized every time the `while` loop is executed, when the loop terminates, `@fullpath` will have the value it had in the last iteration of the loop, which means all other iterations will have no impact whatsoever on its value...This is going to work __only__ if one line is fed to `STDIN`.

Reply:
oops, I should have used @fullpatharray for the looping below, not @fullpath...